pandas/core/arrays/__init__.py

```diff
@@ -1,2 +1,5 @@
 from .base import ExtensionArray  # noqa
 from .categorical import Categorical  # noqa
+from .datetimes import DatetimeArrayMixin  # noqa
+from .periods import PeriodArrayMixin  # noqa
```

Comment:
can you make this `period`? and below timedelta 
I am fine with keeping the conflicting one as plural, if you want that

Reply:
which is "the conflicting one"?

Reply:
Sorry, I was speaking about the file *name*, not the order. So to use singular `period` instead of `periods` (and same for timedelta), as we discussed about before: https://github.com/pandas-dev/pandas/pull/19902/files/3a67bce8169663430005b2a36673132fc1e79f4c#r175774283